infrastructure/prod.tfvars

```diff
@@ -18,4 +18,5 @@ http_client_max_total = 200
 http_client_max_client_per_route = 40
 #http_client_validate_after_inactivity = 0
 
-definition_latest_version_ttl_sec = 5
\ No newline at end of file
+definition_latest_version_ttl_sec = 5
+definition_jurisdiction_ttl_sec = 5
```

Comment:
considering this operation is called only once when a user logs in, I doubt 5 sec ttl will have any effect on performance. It'll help only if 2 users that retrieve the same jurisdictions login max 5 sec apart from each other

Reply:
We can change this number, but the caching of the Jurisdiction list is already serving for purpose of that particular use. Approximately 3K calls per day, this optimisation can still be useful, if not sensible across the entire system.